core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -1191,58 +1193,58 @@ private OptimizelyUserContext createUserContextCopy(@Nonnull String userId, @Non
         return new OptimizelyUserContext(this, userId, attributes, Collections.EMPTY_MAP, null, false);
     }
 
+    Optional<FeatureDecision> getForcedDecision(@Nonnull String flagKey,
+                                                @Nonnull DecisionReasons decisionReasons,
+                                                @Nonnull ProjectConfig projectConfig,
+                                                @Nonnull OptimizelyUserContext user) {
+
+        OptimizelyDecisionContext optimizelyDecisionContext = new OptimizelyDecisionContext(flagKey, null);
+        DecisionResponse<Variation> forcedDecisionVariation = decisionService.validatedForcedDecision(optimizelyDecisionContext, projectConfig, user);
+        decisionReasons.merge(forcedDecisionVariation.getReasons());
+        if (forcedDecisionVariation.getResult() != null) {
+            return Optional.of(new FeatureDecision(null, forcedDecisionVariation.getResult(), FeatureDecision.DecisionSource.FEATURE_TEST));
+        }
+
+        return Optional.empty();
```

Comment:
I understand `Optional` is not supported in all Android versions.
Java-SDK core is used by Android-SDK, so we should stick to Java-8 or old versions old.
Some Java8 features (like Optional) not 100% supported.

Reply:
removed